terraform/gitops/stateful-resources/templates/stateful-resources/namespace.yaml.tpl

```diff
@@ -3,5 +3,14 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: ${ns}
+# tried to implement OR operator but couldn't find a syntax for it.
```

Comment:
@aaronreynoza did you check this https://developer.hashicorp.com/terraform/language/expressions/operators#logical-operators